website/docs/ComponentTesting.md

```diff
@@ -21,9 +21,9 @@ WebdriverIO has built-in support for the following frameworks:
 To set-up WebdriverIO for unit or component testing in the browser, initiate a new WebdriverIO project via:
 
 ```bash
-npm init wdio@latest ./
```

Comment:
Is `npm init wdio@latest ./` still working though? We have now a lot of material out there where we advocate for this command and we should not break these.

Reply:
Furthermore, we own the `wdio` NPM package and can use it to shorten this to `npx wdio`, wdyt?